src/commands/WebsiteCommand.ts

```diff
@@ -0,0 +1,21 @@
+import {Message} from "discord.js";
+import Command from "../abstracts/Command";
+
+class WebsiteCommand extends Command {
+	constructor() {
+		super(
+			"website",
+			"Displays the website with given parameter",
```

Comment:
Can this please be `Displays a link to the CodeSupport's website (or a specific page if specified).`

Reply:
👍